src/ray/util/signal_handler.h

```diff
@@ -0,0 +1,36 @@
+#ifndef RAY_UTIL_SIGNAL_HANDLER_H
+#define RAY_UTIL_SIGNAL_HANDLER_H
+
+#include <string>
+#include <vector>
+
+namespace ray {
+
+class SignalHandler {
```

Comment:
`class SignalHandlers` ?

Reply:
Because all of the methods are static, maybe delete ctor and dtor will be better?